### PR TITLE
Update to use nest ids

### PR DESCRIPTION
**Pull request by petergaemers**
Update to use nest ids to refer to S1 scintilation models, note only ER actually makes sense.
Also change detector field name to accomidate epix and placeholder functions to strax_interface

### CODE REVIEW

File: wfsim/strax_interface.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def rand_instructions(c):
     instructions['event_number'] = np.digitize(instructions['time'],
          1e9 * np.arange(c['nchunk']) * c['chunk_size']) - 1
     instructions['type'] = np.tile([1, 2], n)
-    instructions['recoil'] = [1 for i in range(n * 2)]
+    instructions['recoil'] = [7 for i in range(n * 2)] #Use nest ids for  ER
```
Comment (ramirezdiego) [inline wfsim/strax_interface.py:52]: Is it too late to comment here? There is no error, but let's just remember to document somewhere that the rand_instructions produce gammas only.

Comment (jorana) [reply wfsim/strax_interface.py:52]: you can also just `from nestpy.helpers import NEST_INTERACTION_NUMBER` and be done with it :) 
https://github.com/NESTCollaboration/nestpy/blob/master/src/nestpy/helpers.py#L27

Comment (jorana) [reply wfsim/strax_interface.py:52]: also it's written down here:
https://github.com/XENONnT/WFSim/blob/master/wfsim/core.py#L22

Comment (ramirezdiego) [reply wfsim/strax_interface.py:52]: I don't understand what you mean in the second comment.

Comment (jorana) [reply wfsim/strax_interface.py:52]: the way I phrased it not correct so I understand you don't understand 😄 . Never-mind, it wouldn't hurt to indeed explicitly add in the NestId what 7,8,11 are like you mention.

**Pull request merged**